includes/Hooks.php

```diff
@@ -96,16 +97,39 @@ public function onSpecialPageBeforeExecute( $special, $subPage ) {
 			return true;
 		}
 
+		$config = MediaWikiServices::getInstance()->getMainConfig();
+		$protectedSpecialPages = $config->get( 'CrawlerProtectedSpecialPages' );
+		$denyFast = $config->get( 'CrawlerProtectedSpecialPages' );
+
 		$name = strtolower( $special->getName() );
-		if ( in_array( $name, [ 'recentchangeslinked', 'whatlinkshere' ], true ) ) {
+		if (
+			// allow forgiving entries in the setting array for Special pages names
+			in_array( $special->getName(), $protectedSpecialPages, true )
```

Comment:
These 3 lines will be redundant once the transformation is applied. Please remove the two extra lines